wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+  - **...entre 2:30 y 3:15**, la dificultad más baja de cada modo de tecla incluido no puede ser más difícil que una dificultad Difícil, **O** cada modo de tecla tiene que proveer 2 dificultades más bajas debajo de la dificultad más difícil.
+  - **...entre 3:15 y 4:00**, la dificultad más baja de cada modo de tecla incluido no puede ser más difícil que una dificultad Insana, **O** cada modo de tecla tiene que proveer 1 dificultad más baja debajo de la dificultad más difícil.
+- En las dificultades Insanas o más difíciles, las dificultades apropiadamente hechas deben tener una brecha de dificultad similar a las que se formalmente definieron en las dificultades bajas.
+
+### Pautas
+
+- **Cada nota debe relacionarse a un sonido presente en la música.** Esto puede ser un sonido distintivo, pero también puede representar coherentemente un sonido continuo que no tiene un principio of final distinguible. Excepciones a favor de la simplificación deben ser hechas cuando se siga la música precisamente...
+  - no sería razonable de jugar con buena precisión.
+  - seria efectivamente representando con un divisor constante y predecible.
+  - causaría un pico inapropiado en la dificultad.
+- **Las notas largas deben ser presionadas por al menos 1/12 de una pulsación.** Tiempos de agarre más rápido no son razonables para poder obtener completa precisión.
+- **Evite usar hitsound sampleados en el storyboard con un volumen similar a los sonidos de las notas.** Como los hitsound pueden agobiar y distorsionar la retroalimentación de audio de los objetos en el beatmap, poner sonidos encima de la nota obtiene prioridad. Sin embargo, samples del storyboard pueden ser agregados como keysound si son extraídos directamente de la canción y cuando las dificultades bajas tienen una poca cantidad de notas para soportar los hitsound.
+- **No utilice keysounds sin un hitnormal como soporte.** Si los hitsounds o hitsound del storyboard se mezclan perfectamente con la canción, la retroalimentación para estas notas es mínima. Sin embargo, cuando los keysounds son extraídos directamente de la canción y usados como un reemplazo del audio, el hitnormal no es requerido.
+- **Evite usar el mismo hitsound para múltiples notas en un chord.** Esto causa que la retroalimentación del audio se más alta de lo necesario. Esto también aplica para los hitsounds activos y storyboard hitsounds usados en la misma marca de tiempo.
+- **Los cambios de tiempo deben tener una velocidad de desplazamiento unificada ajustando la velocidad del slider.** Esto se hace normalizando la velocidad de desplazamiento de todas las secciones con cambios de tiempo a la que más se usa en el beatmap. Este nombre es mostrado en paréntesis después del rango de BPM en el menú de selección ej. `BPM: 60-160 (130)`. Tener en cuenta que esta pauta no afecta la habilidad de agregar cambios de velocidades del slider que son usados para crear efectos de SV.

```

Comment:
```suggestion
- **Los cambios de tiempo deben tener una velocidad de desplazamiento unificada ajustando la velocidad del slider.** Esto se hace normalizando la velocidad de desplazamiento de todas las secciones de timing a la más utilizada en el beatmap. El número se muestra entre paréntesis después del rango de BPM en la selección de la canción, por ejemplo, `BPM: 60-160 (130)`. Tenga en cuenta que esta pauta no afecta a la posibilidad de añadir cambios de velocidad del slider que se utilizan para crear efectos SV.
```

Reply:
Reworded this differently due to the use of the word timing